### PR TITLE
ci: test simd-accel feature

### PR DESCRIPTION
**Pull request by BurntSushi**
This is in response to:
https://github.com/hsivonen/encoding_rs/issues/64#issuecomment-720505594

### CODE REVIEW

**Comment by BurntSushi** [comment]
I'm not sure why CI isn't running..? Weird.

File: .travis.yml [hunk 1]
```diff
@@ -15,3 +15,8 @@ matrix:
     rust: nightly
     script:
       - sh ci/miri.sh
+  - name: simd
+    rust: nightly
+    script:
+      - sh ci/miri.sh
```
Comment (hsivonen) [inline .travis.yml:21]: Is this line intentional?

Comment (BurntSushi) [reply .travis.yml]: Ah nope, good catch! Fixed.

**Comment by hsivonen** [comment]
Thanks!

**Pull request merged**